mapie/control_risk/p_values.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+from numpy.typing import NDArray
+from typing import Iterable, Union
+from scipy.stats import binom
+
+
+def hoefdding_bentkus_p_value(
+    r_hat: NDArray,
+    n: int,
+    alpha: Union[float, NDArray]
+) -> NDArray:
+    """
+    The method computes the p_values according to
+    the Hoeffding_Bentkus inequality for each
+    alpha.
+    We return the minimum between the Hoeffding and
+    Bentkus p-values (Note that it depends on
+    scipy.stats). The p_value is introduce in 
+    learn then test paper.
+
+    References
+    ----------
+    [1] Angelopoulos, A. N., Bates, S., Candès, E. J., Jordan,
+    M. I., & Lei, L. (2021). Learn then test:
+    "Calibrating predictive algorithms to achieve risk control".
+
+    Parameters
+    ----------
+    r_hat: NDArray of shape (n_lambdas, )
+        Empirical risk of metric_control with respect
+        to the lambdas.
+        Note: r_hat is the empirical mean of a matrix of
+        shape (n_samples, n_lambdas).
+
+    n: Integer value
+        Correspond to the number of observations in
```

Comment:
QUESTION: is `n=n_samples`? Because it is not clear here.

Reply:
Yes

Reply:
Add this precision in the docstring please.

Reply:
Remplace `n` by `n_obs` for clarity.